src/proxy/server.ts

```diff
@@ -37,12 +37,14 @@ if (!isHttpUrl(SWA_CLI_AUTH_URI)) {
   process.exit(-1);
 }
 
+// make SWA assets referencable
 const SWA_404 = path.resolve(process.cwd(), "..", "public", "404.html");
 const SWA_401 = path.resolve(process.cwd(), "..", "public", "unauthorized.html");
+const htmlAssets: Record<string, string> = { SWA_404, SWA_401 };
 
 const serveStatic = (file: string, res: http.ServerResponse, status = 200) => {
   if (status !== 401) {
-    file = fs.existsSync(file) ? file : SWA_404;
+    file = fs.existsSync(file) ? file : htmlAssets["SWA_404"];
```

Comment:
Added configuration object, to make the static responses dynamically addressable.